internal/pkg/runtime/engines/imgbuild/process_linux.go

```diff
@@ -25,13 +25,13 @@ func (e *EngineOperations) StartProcess(masterConn net.Conn) error {
 
 	if e.EngineConfig.RunSection("post") && e.EngineConfig.Recipe.BuildData.Post.Script != "" {
 		// Run %post script here
-		e.runScriptSection("post", e.EngineConfig.Recipe.BuildData.Post, true)
+		e.runScriptSection(nil, "post", e.EngineConfig.Recipe.BuildData.Post, true)
```

Comment:
I understand why this is, but please add a comment indicating that the nil is intentional and the reason for it.